packages/design-system/src/components/Table/__snapshots__/Table.test.jsx.snap

```diff
@@ -45,80 +15,12 @@ exports[`Table applies responsive table 1`] = `
 </div>
 `;
 
-exports[`Table applies scroll table 1`] = `
```

Comment:
@nichia Where is this snapshot from? It look important to keep

Reply:
Yes, that snapshot has been added back.